src/DependencyInjection/DependencyInjection/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("CodeMonkeys.UnitTests")]
+[assembly: InternalsVisibleTo("CodeMonkeys.DependencyInjection.Ninject")]
+[assembly: InternalsVisibleTo("CodeMonkeys.DependencyInjection.DryIoC")]
+namespace CodeMonkeys.DependencyInjection
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty namespace.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)